templates/test/tr_job_result_details.html.ep

```diff
@@ -0,0 +1,7 @@
+% if ($comments and $comments > 0) {
```

Comment:
previous sets $comments to the number of comments, right? if ($comments) will be false if that number is 0. So where is the problem?